src/core/common/message.hpp

```diff
@@ -798,6 +800,14 @@ class Message : public Buffer
      */
     const RssAverager &GetRssAverager(void) const { return GetMetadata().mRssAverager; }
 
+    /**
+     * This method sets the link info fields (PAN ID, link security, RSS) from a given `ThreadLinkInfo`.
+     *
+     * @param[in] aLinkInfo   A Thread Link Info.
```

Comment:
```suggestion
     * @param[in] aLinkInfo   The Thread Link info.
```
Not sure if we should use `Link` or `link`. But I think we should be consistent in Doxygen.